.github/workflows/ci.yml

```diff
@@ -257,6 +257,29 @@ jobs:
         run: docker logs earthly-buildkitd
         if: ${{ failure() }}
 
+  export-test:
+    if: github.event_name == 'push' || github.event.pull_request.head.repo.full_name == github.repository
```

Comment:
This test seems excluded for fork PRs - is this necessary?

Reply:
i'll remove it; just a copy and past from one of our excluded tests